src/btest.cr

```diff
@@ -229,12 +231,14 @@ class Case
 
       res = Process.run(cmd2, nil, shell: true, output: stdout, error: stderr)
 
-      if !res.success?
-        Result.new(runner, self, false,
-          "Error running: #{cmd2}\n" \
-          "Status code: #{res.exit_code}\n" \
-          "Standard output: #{stdout.to_s}\n" \
-          "Standard error: #{stderr.to_s}\n")
+      unless res.success?
+        Result.new(runner, self, false, <<-OUTPUT
+        Error running: #{cmd2}
```

Comment:
Could you indent a bit here please? It feels easier to follow when it's indented